app/views/orders/new.html.haml

```diff
@@ -28,45 +28,73 @@
         = link_to "cool perks", "#perks"
         in return!
 
-  .row
-    .col-md-6.col-xs-12.order-form
-      = render 'form'
-
-    .col-md-6.col-xs-12
-
-      %p
-        What do you get with premium?
-
-      = render "shared/perks"
-
-      %p#perks
-        We use the money for the following things:
-        %ul
-          %li
-            Pay
-            = link_to("the server providers", server_providers_path)
-            their share, based on minutes played on their servers
-          %li Pay for the hosting of the site
-          %li Fund additional gameservers
-
-      %p
-        Premium status is site-specific, so you either pay for serveme.tf, na.serveme.tf or sea.serveme.tf, make sure you're on the right site before paying.
-
-      %p
-        You can also
-        - if sea_system?
-          = link_to('send the tradebot a trade offer.', 'https://steamcommunity.com/tradeoffer/new/?partner=328658734&token=3vDIZUwa')
+  - if au_system?
+    .row
+      .col-md-12
+
+        %p
+          What do you get with premium?
+
+        = render "shared/perks"
+
+        %p#perks
+          We use the money for the following things:
+          %ul
+            %li
+              Pay
+              = link_to("the server providers", server_providers_path)
+              their share, based on minutes played on their servers
+            %li Pay for the hosting of the site
+            %li Fund additional gameservers
+
+        %p
+          Premium status is site-specific, so you either pay for serveme.tf, na.serveme.tf, sea.serveme.tf or au.serveme.tf, make sure you're on the right site before paying.
+
+        %p
+          Get
+            = link_to('au.serveme.tf premium by subscribing to the ozfortress Patreon.', 'https://patreon.com/ozfortress')
```

Comment:
This line is not valid (nesting in plain text). Should just be a matter of putting this line at the same start position as the line above it.